tests/integration/change/change.spec.ts

```diff
@@ -0,0 +1,127 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */ // some vars are not really needed
+/* eslint-disable jest/no-conditional-expect */ //needed for testing all action cases in one place
```

Comment:
1. for unused var please update your @map-colonies/eslint-config
2. remove no-uncoditional-expect - better to unroll the tests than using conditional-expects

Reply:
done and done.